batch-indexing/src/main/java/data/processor/IndexingItemProcessor.java

```diff
@@ -0,0 +1,76 @@
+package data.processor;
+
+import data.entity.ExecutionRecord;
+import data.entity.ExecutionRecordDTO;
+import data.job.BatchJobType;
+import data.utility.ExecutionRecordUtil;
+import eu.europeana.indexing.IndexerFactory;
+import eu.europeana.indexing.IndexingSettings;
+import eu.europeana.indexing.tiers.model.MediaTier;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.batch.core.configuration.annotation.StepScope;
+import org.springframework.batch.item.ItemProcessor;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+import java.lang.invoke.MethodHandles;
+import java.time.Instant;
+import java.util.Collections;
+import java.util.Date;
+import java.util.Optional;
+
+@Component
+@StepScope
+public class IndexingItemProcessor implements ItemProcessor<ExecutionRecord, ExecutionRecordDTO> {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(MethodHandles.lookup().lookupClass());
+
+    @Value("#{jobParameters['PRESERVE_TIMESTAMPS']}")
```

Comment:
All other parameters are camel cased and identical to the java parameter name.
I would suggest we keep it that way here too.

Reply:
Done